examples/app/BUILD.bazel

```diff
@@ -1,22 +1,108 @@
-load("@build_bazel_rules_typescript//:defs.bzl", "ts_devserver", "ts_library")
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+package(default_visibility = ["//visibility:public"])
+
+load("@build_bazel_rules_typescript//:defs.bzl", "ts_library", "ts_devserver")
+load("@io_bazel_rules_closure//closure:defs.bzl", "closure_js_library", "closure_js_binary")
+load("//internal:build_defs.bzl", "closure_ts_binary")
+load("//examples/app:internal_node_module.bzl", "internal_node_module")
 
 ts_library(
-    name = "app",
-    srcs = ["app.ts"],
+    name = "ts_library",
+    srcs = ["main.ts"],
+    deps = ["//examples/app/hello-world"],
+)
+
+# Needed because the devserver only loads static files that appear under this
+# package.
+genrule(
+    name = "angular_js",
```

Comment:
Why should the canonical typescript app example use Angular? We should be careful about layering, this rule is generally useful to all typescript users